yb-voyager/src/srcdb/srcdb.go

```diff
@@ -42,8 +42,8 @@ type SourceDB interface {
 	GetCharset() (string, error)
 	FilterUnsupportedTables(migrationUUID uuid.UUID, tableList []*sqlname.SourceName, useDebezium bool) ([]*sqlname.SourceName, []*sqlname.SourceName)
 	FilterEmptyTables(tableList []*sqlname.SourceName) ([]*sqlname.SourceName, []*sqlname.SourceName)
-	GetColumnsWithSupportedTypes(tableList []*sqlname.SourceName, useDebezium bool, isStreamingEnabled bool) (map[*sqlname.SourceName][]string, []string)
-	GetTableColumns(tableName *sqlname.SourceName) ([]string, []string, []string)
+	GetColumnsWithSupportedTypes(tableList []*sqlname.SourceName, useDebezium bool, isStreamingEnabled bool) (map[*sqlname.SourceName][]string, map[*sqlname.SourceName][]string, error)
+	GetTableColumns(tableName *sqlname.SourceName) ([]string, []string, []string, error)
```

Comment:
I see that there is no caller of this function. I think you can get rid of it from this interface.